weasyl/templates/common/footer.jinja2

```diff
@@ -0,0 +1,72 @@
+<footer>
+    <div id="footer">
+        <nav>
+            <div id="footer-nav" class="clear">
+                <div class="column clear">
+                    <h1><a href="/index"><img src="{{ 'img/logo.png' | route_path }}" alt="Weasyl"/></a></h1>
+                    <ul class="clear">
+                        <li><a href="{{ 'submit' | route_path }}">Submit</a></li>
+                        <li><a href="{{ 'search' | route_path }}">Search</a></li>
+                        <li><a href="{{ 'control' | route_path(alias='control') }}">Settings</a></li>
+                    </ul>
+                </div>
+                <div class="column clear">
+                    <h2 class="typeface-a color-a">About</h2>
+                    <ul class="clear">
+                        <li><a href="{{ 'help_about' | route_path }}">About Weasyl</a></li>
+                        <li><a href="{{ 'site_update_list' | route_path }}">News</a></li>
+                        <li><a href="{{ 'help_faq' | route_path }}">FAQ</a></li>
+                        <li><a href="{{ 'staff' | route_path }}">Staff</a></li>
+                        <li><a href="{{ 'thanks' | route_path }}">Awesome People</a></li>
+                    </ul>
+                </div>
+                <div class="column clear">
+                    <h2 class="typeface-a color-a">Community</h2>
+                    <ul class="clear">
+                        <li><a href="https://forums.weasyl.com/vbulletin/forum.php">Forums</a></li>
```

Comment:
Changes to the footer need to be merged in.

Reply:
I missed the email changes in it too.
Fixed :)